sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -382,7 +381,7 @@ export class ManagementClient extends LinkEntity<RequestResponseLink> {
    */
   async peek(
     messageCount: number,
-    options?: OperationOptionsBase & SendManagementRequestOptions
+    options: SendManagementRequestOptions = {}
```

Comment:
Do you need this default arg? I looked, peek() is only ever called with options filled out.

Reply:
Done.